integration-test/src/main/java/com/sequenceiq/it/cloudbreak/dto/distrox/DistroXTestDto.java

```diff
@@ -336,7 +336,13 @@ public Clue investigate() {
         boolean hasSpotTermination = (getResponse().getInstanceGroups() == null) ? false : getResponse().getInstanceGroups().stream()
                 .flatMap(ig -> ig.getMetadata().stream())
                 .anyMatch(metadata -> InstanceStatus.DELETED_BY_PROVIDER == metadata.getInstanceStatus());
-        return new Clue("DistroX", auditEvents, getResponse(), hasSpotTermination);
+        return new Clue(
+                "DataHub",
```

Comment:
Can we append or replace this with the name of the resource? crn contains information about the resource and I feel like this is overlapping.

Reply:
Yes sure. I'll change it.